docs/tables/{custom_resource_name.group_name}.md

```diff
@@ -0,0 +1,556 @@
+                  type: string
+                  format: date-time
+                nextPrivateKeySecretName:
+                  description: The name of the Secret resource containing the private key to be used for the next certificate iteration. The keymanager controller will automatically set this field if the `Issuing` condition is set to `True`. It will automatically unset this field when the Issuing condition is not set or False.
+                  type: string
+                notAfter:
+                  description: The expiration time of the certificate stored in the secret named by this resource in `spec.secretName`.
+                  type: string
+                  format: date-time
+                notBefore:
+                  description: The time after which the certificate stored in the secret named by this resource in spec.secretName is valid.
+                  type: string
+                  format: date-time
+                renewalTime:
+                  description: RenewalTime is the time at which the certificate will be next renewed. If not set, no upcoming renewal is scheduled.
+                  type: string
+                  format: date-time
+                revision:
+                  description: "The current 'revision' of the certificate as issued. \n When a CertificateRequest resource is created, it will have the `cert-manager.io/certificate-revision` set to one greater than the current value of this field. \n Upon issuance, this field will be set to the value of the annotation on the CertificateRequest resource used to issue the certificate. \n Persisting the value on the CertificateRequest resource allows the certificates controller to know whether a request is part of an old issuance or if it is part of the ongoing revision's issuance by checking if the revision value in the annotation is greater than this field."
+                  type: integer
+      served: true
+      storage: true
+```
+
+```bash
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: temporal-w-spcloudt6t6sk7toegg-tls
+  labels:
+    workspace-id: w_spcloudct6t6sk7toegg
+    identity-id: o_spcloud6un5ogrh1bc0
+    region: apse1
+    shard: "0001"
+    workspace-pluginset: "20220317010004"
+    app.kubernetes.io/component: steampipe-workspace-db
+    app.kubernetes.io/managed-by: steampipe-api
+    app.kubernetes.io/part-of: steampipe-cloud
+    app.kubernetes.io/instance: w-spcloudct6t6sk7toegg
+    app.kubernetes.io/version: 0.13.3-workspace-spcloud.20220317010004
+spec:
+  secretName: temporal-w-spcloudt6t6sk7toegg-tls
+  duration: 87600h # 10 years
+  renewBefore: 360h # 15d
+  usages:
+    - server auth
+    - client auth
+  issuerRef:
+    name: ca-issuer-temporal
+    kind: ClusterIssuer
+  dnsNames:
+    - w-spcloudt6t6sk7toegg
+```
+
```

Comment:
Can you please add sections like in https://hub.steampipe.io/plugins/turbot/prometheus/tables/%7Bmetric_name%7D, starting with: `This plugin will automatically create a table called prometheus_http_requests_total:`